VicBikeMap/settings/base.py

```diff
@@ -14,20 +14,20 @@
 import os
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
 
-MIDDLEWARE_CLASSES = (
+MIDDLEWARE = (
     'corsheaders.middleware.CorsMiddleware',
     'django.middleware.gzip.GZipMiddleware',
     'debug_toolbar.middleware.DebugToolbarMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
     # 'middlewares.ForceDefaultLanguageMiddleware',
-    'solid_i18n.middleware.SolidLocaleMiddleware',
+    # 'solid_i18n.middleware.SolidLocaleMiddleware',
     # 'django.middleware.locale.LocaleMiddleware',
     'django.middleware.common.CommonMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
-    'minidetector.Middleware',
+    'django_user_agents.middleware.UserAgentMiddleware',
```

Comment:
Since Minidetector is not supported in python3 we will be using `UserAgentMiddleware` for mobile compatibilty